src/index.html

```diff
@@ -13,6 +13,42 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      data-qa="card"
+      class="card"
+    >
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="imac-logo"
+      />
+      <div class="card__name-panel">
+        <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      </div>
+      <div class="card__product-code">
+        <p>Product code: 195434</p>
+      </div>
+      <div class="card__review">
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p>Reviews: 5</p>
+      </div>
+      <div class="card__price-panel">
+        <p class="card__price-panel-text">Price:</p>
+        <p class="card__price-panel-value">$2,199</p>
+      </div>
+      <a
```

Comment:
Add the blank line between the blocks
```suggestion
      </div>
      
      <div class="card__price-panel">
        <p class="card__price-panel-text">Price:</p>
        <p class="card__price-panel-value">$2,199</p>
      </div>
      
      <a
```